app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_one :user
+
```

Comment:
- [ ] Please update the associations as below because a post also has many comments and likes.

 ```ruby
  belongs_to :author, class_name: 'User'
  has_many :likes
  has_many :comments